src/graph/graph_type.c

```diff
@@ -57,6 +57,8 @@ void _GraphType_LoadMatrices(RedisModuleIO *rdb, Graph *g) {
         GrB_Matrix m = Graph_GetLabel(g, matrixIdx);
         _GraphType_LoadMatrix(rdb, m);
     }
+    // Flush all pending changes to graphs
+    GrB_wait();
```

Comment:
Not strictly necessary, but I think this is a sensible place for this call, which will obviate some of the work performed by _Graph_ResizeMatrix (when loaded, all values might be pending).

Reply:
👍